spec/support/mongo_helper.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module MongoHelper
+  def mongo_config(opts = {})
+    prefix = ENV.fetch('TEST_ENV_NUMBER', '')
+    opts.merge(collection: "#{prefix}#{opts[:collection]}")
+  end
+
+  def self.mongo_config(args)
+    Class.include(MongoHelper).new.mongo_config(args)
+  end
+end
```

Comment:
To follow the existing convention, this file should be located at `spec/moneta/adapters/mongo/helper.rb`, and required from the specs that need it, instead of from the `spec/helper.rb` file.  Is there a reason you have opted to do it this way?

Reply:
Also, do you have some sense of why we suddenly need this?  Was there a change to how MongoDB works?

Reply:
Actually, I've thought about this some more and I like your approach here better than having the helpers co-located with the specs, so that's a :+1: from me for this change.